src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,43 @@
 using namespace std;
 
 int t06_cinema() {
+    int n, m;
+    cin >> n >> m;
+    int a[n];
+    int k;
+    int prev = -1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'prev' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493528)